psasim/test/run_test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This is a simple bash script that tests psa_client/psa_server interaction.
+# This script is automatically executed when "make run" is launched by the
+# "psasim" root folder. The script can also be launched manually once
+# binary files are built (i.e. after "make test" is executed from the "psasim"
+# root folder).
+#
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+set -e
+
+function clean_run() {
+    pkill psa_partition || true
+    pkill psa_client || true
+    ipcs | grep q | awk '{ printf " -q " $$2 }' | xargs ipcrm > /dev/null 2>&1 || true
+}
+
+clean_run
+
+./psa_partition &
```

Comment:
This test should run the server in such a way that it terminates when the client disconnects, to leave this script self-contained (i.e. not leave the server running after the client, with the script waiting)

Reply:
Sorry, I changed the behavior of the "k" parameter recently, but then I forgot to update this launch command.